authors/apps/comments/views.py

```diff
@@ -0,0 +1,236 @@
+from authors.apps.articles.models import Article
+from .serializers import CommentsSerializers
+from .models import Comments
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework.generics import (RetrieveUpdateDestroyAPIView,
+                                     ListCreateAPIView)
+
+
+class CommentsAPIView(APIView):
+    """
+    Creates a CRUD view for all comments
+    """
+    permission_classes = (IsAuthenticatedOrReadOnly,)
+    serializer_class = CommentsSerializers
+
+    def get(self, request, **kwargs):
+        slug = self.kwargs['slug']
+        article = Article.objects.filter(slug=slug).first()
+
+        if not article:
+            return Response(
+                {
+                    "comment": {
+                        "error": "Sorry, article was not found"
+                    }
+                },
+                status=status.HTTP_204_NO_CONTENT
+            )
+
+        queryset = Comments.objects.filter(article=article.id)
+        if not queryset:
+            return Response(
+                {
+                    "comment": {
+                        "error": "Sorry, You do not have any comments yet"
+                    }
+                },
+                status=status.HTTP_204_NO_CONTENT
+            )
+        serializer = self.serializer_class(data=queryset, many=True)
+        serializer.is_valid()
+        return Response({
+            "comment": serializer.data,
+            "commentsCount": queryset.count()
+        },
+            status=status.HTTP_200_OK
+        )
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    def post(self, request, **kwargs):
+        """
+        Method for creating comments
+        """
+        slug = self.kwargs['slug']
+        article = Article.objects.filter(slug=slug).first()
+        if not article:
+            return Response(
+                {
+                    "comment": {
+                        "error": "Sorry, article was not found"
+                    }
+                },
+                status=status.HTTP_204_NO_CONTENT
+            )
+        comment = request.data
+        serializer = self.serializer_class(data=comment, partial=True)
+        serializer.is_valid(raise_exception=True)
+        serializer.save(article=article, author=request.user)
+        return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+
+class CommentDetailsAPIView(RetrieveUpdateDestroyAPIView, ListCreateAPIView):
+    """
+    Creates a view to Get, update, and delete a particular comment
+    """
+
+    permission_classes = (IsAuthenticatedOrReadOnly,)
+    serializer_class = CommentsSerializers
+
+    def get(self, request, id, *args, **kwargs):
```

Comment:
I suggest you do away with the `*args` argument is not used anywhere thus not necessary.